README.md

```diff
@@ -45,25 +45,24 @@ Download and unzip the latest version of Avatar Animator from the [release page]
+| Zoom            | Go to `Settings` -> `Video` and select `OBS-Camera` from the `Camera` drop-down menu.                                                         | <img src="./images/zoom.png" alt="Avatar Animator on Zoom: configuration" width="600">                |
+| Skype           | Go to `Settings` -> `Audio & Video` and select `OBS-Camera` from the `Camera` drop-down menu.                                                 | TODO                                                                                                  |
+| Microsoft Teams | Click on your profile picture at the top of Teams, then `Settings` -> `Devices` and select `OBS-Camera` from the `Camera` drop-down menu.     | <img src="./images/ms-teams.png" alt="Avatar Animator on Microsoft Teams: configuration" width="600"> |
+| Google Meet     | Start a meeting, then click `Settings` > `General` and select `OBS-Camera` from the `Camera` drop-down menu.                                  | <img src="./images/google-meet.png" alt="Avatar Animator on Google Meet: configuration" width="600">  |
+| Slack           | Start a call, then click on the gear icon in the top left corner of the call window and select `OBS-Camera` from the `Camera` drop-down menu. | TODO                                                                                                  |
 
 
 ## Configuration
 
-The configuration panel (top right in Avatar Animator; figure below) allows the configuration of cameras, avatars and background colours, and display of debug information. To import new avatars or a background images, drag and drop them into Avatar Animator (note that new avatars will not be added to the `avatar` drop down menu). To create a new avatar, follow the instructions as detailed in [AVATARS.md](AVATARS.md).
+The configuration panel (top right in Avatar Animator; figure below left) allows the **choice of camera, avatar, background color, and display of debug information**. To **add a new background image**, drag and drop it into Avatar Animator. To **create a new avatar or modify and exiting one**, see [AVATARS.md](AVATARS.md), then to **add a new avatar** drag and drop it into Avatar Animator (note that new avatars will not be added to the `avatar` drop down menu).
 
-<div align="center">
-    <img src="./images/control-panel.png" alt="Avatar Animator control panel" width="200">
-    <br>
-    <b>Avatar Animator control panel.</b>
-</div>
+In the latest versions of Avatar Animator, a simplified avatar editor (figure below right) is also available. This allows to **modify anchor (skeleton) points and colors** for existing avatars. Currently, when an anchor point is dragged (only those are modifiable), the position of any name-matching illustration anchor points are set to the same position.
 
+| <img src="./images/control-panel.png" alt="Control panel" height="395"> | <img src="./images/editor.png" alt="Editor" height="395">            |
```

Comment:
What about the editor screenshot? Don't you think it would be better to make that smaller in width?

Reply:
see https://github.com/letmaik/avatar-animator/pull/29/commits/db8e6fd22b08fab8bba53929737b58ab2158a0fa

Reply:
You only resized the screenshot. What I meant was take a new screenshot where you resized the window width to avoid the white space. Does that make sense?